packages/passenger/rpm/passenger.spec

```
@@ -111,6 +114,9 @@ memory usage. Includes Phusion Passenger support.
 %setup -q -n %{pkg_name}-%{passenger_version}
 %setup -q -T -D -a 1 -n %{pkg_name}-%{passenger_version}
 
+# Apply patches
+%patch -P0 -p1 -F1
```

```suggestion
%patch0 -p1 -F1
```
I made this change also for 4.0 builds since I didn't notice this on previous review.  I'm more familiar with `%patchN` notation when not doing auto patch also it means the patch command has same flags for RPM and Deb, ie `-p1 -F1`.

---

Fedora 42 seems to use RPM 4.20, so I guess RHEL 11 would be incompatible when it releases in 2027 :smile: 

I used this site as source: https://fedora.pkgs.org/42/fedora-x86_64/rpm-4.20.1-1.fc42.x86_64.rpm.html

---

Interesting, then fine to keep and I'll put it back for 4.0 branch.  Thanks for the details.